src/raftstore/store/snap.rs

```diff
@@ -171,13 +150,25 @@ impl SnapFile {
         Ok(())
     }
 
+    /// Same as `raw_save`, but will automatically append the checksum to
+    /// the end of file.
+    pub fn save(&mut self) -> io::Result<()> {
+        self.save_impl(true)
+    }
+
     /// Use the content in temporary files replace the target file.
     ///
     /// Please note that this method can only be called once.
-    pub fn save(&mut self) -> io::Result<()> {
+    pub fn raw_save(&mut self) -> io::Result<()> {
```

Comment:
FYI, I prefer to use
save -> nothing more than just flush and close the file.
save_with_checksum -> a longer name compared to 'save', which means more work to do: to append checksum, flush and close file.